bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WWZJets_4f_NLO/WWZJets_4f_NLO_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+#import model loop_sm-ckm
+#switch to diagonal ckm matrix if needed for speed
+import model loop_sm
+
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define w = w+ w-
+
+# exclude s-channel top contribution to avoid
+# overlap with TTZ sample
+generate p p > w w z $$ t t~ [QCD] @0
```

Comment:
since you are using 4 flavor, there should be no overlap with ttz. another question is about the w/z decay, you will decay them through pythia, right

Reply:
Correct!

Reply:
The W's with this syntax, like Meng said, will be produced on-shell with a delta function in place of the actual width. Using madspin for decays could be a good idea.

Reply:
I can ask the SMP conveners if they want to do that. This is a repeat of the samples from 13 TeV and we have not received any recommendations or requests for a change.